pkg/controllers/routing/network_routes_controller.go

```diff
@@ -985,8 +985,13 @@ func (nrc *NetworkRoutingController) startBgpServer(grpcServer bool) error {
 	}
 
 	if grpcServer {
-		nrc.bgpServer = gobgp.NewBgpServer(
-			gobgp.GrpcListenAddress(nrc.nodeIP.String() + ":50051" + "," + "127.0.0.1:50051"))
+		var addr string
+		if nrc.isIpv6 {
+			addr = "[" + nrc.nodeIP.String() + "]:50051" + "," + "[::1]:50051"
```

Comment:
Random question by the way. Why does this need to listen on `nodeIP` on the first place; and not just on loopback?  `nodeIP` is definitely publicly routable in ipv6 and i'm curious why we're having to spawn a publicly accessible grpc server. Do the different `kube-router` daemonsets connect to eachother over grpc or something?